src/Facebook/InstantArticles/Elements/Element.php

```diff
@@ -37,21 +37,11 @@ public function render($doctype = '', $formatted = false)
         $document->appendChild($element);
         $rendered = $doctype.$document->saveXML($element);
 
-        return $rendered;
-    }
+        // We can't currently use DOMDocument::saveHTML, because it doesn't produce proper HTML5 markup, so we have to strip CDATA enclosures
+        // TODO Consider replacing this workaround with a parent class for elements that will be rendered and in this class use the `srcdoc` attribute to output the (escaped) markup
+        $rendered = preg_replace('/<!\[CDATA\[(.*?)\]\]>/', '$1', $rendered);
```

Comment:
I tried this and not working but `$rendered = preg_replace('/<!\[CDATA\[(.*?)\]\]>/is', '$1', $str);` works for me.